markdowns/19_Text_to_Image_search_pipeline_with_MultiModal_Retriever.md

```diff
@@ -0,0 +1,190 @@
+---
+layout: tutorial
+colab: https://colab.research.google.com/github/deepset-ai/haystack-tutorials/blob/main/tutorials/19_Text_to_Image_search_pipeline_with_MultiModal_Retriever.ipynb
+toc: True
+title: "Text-to-Image search pipeline with MultiModal Retriever"
+last_updated: 2022-11-03
+level: "intermediate"
+weight: 95
+description: Use a MultiModalRetriever to build a cross modal search pipeline.
+category: "QA"
+aliases: ['/tutorials/multimodal']
+---
+    
+
+**Level**: Intermediate
+
+**Time to complete**: 20 minutes
+
+**Prerequisites**: Prepare the Colab environment (see links below).
+
+**Nodes Used**: InMemoryDocumentStore, MultiModalRetriever
+
+**Goal**: After completing this tutorial, you will have learned about the MultiModalRetriever, and built a simple retrieval pipeline that searches for relevant images given a text query.
```

Comment:
How about simplifying it a bit: 
After completing this tutorial, you will have built a search system that retrieves images as answers to a text query.

Reply:
Changed.